bot/seasons/valentines/lovecalculator.py

```diff
@@ -0,0 +1,119 @@
+import logging
+from random import randint
+
+import discord
+from discord.ext import commands
+
+
+logger = logging.getLogger(__name__)
+
+
+LOVE_STAT = {
```

Comment:
These constants would probably be better in a JSON. #105 uses a `bot/resources/valentines/` directory, which will work fine for any static resources you need to add.